src/main/java/bio/terra/app/controller/DataRepositoryServiceApiController.java

```diff
@@ -107,7 +110,17 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
         @RequestParam(value = "expand", required = false, defaultValue = "false") Boolean expand) {
         // The incoming object id is a DRS object id, not a file id.
         AuthenticatedUserRequest authUser = getAuthenticatedInfo();
-        return new ResponseEntity<>(drsService.lookupObjectByDrsId(authUser, objectId, expand), HttpStatus.OK);
+        DRSObject drsObject = null;
+        HttpStatus status = HttpStatus.OK;
+        drsService.increment();
```

Comment:
Is there a reason you decided to put the increment/decrement out here instead of inside of the service?
Seems like they could be inside and not exposed.

Reply:
It seemed like there wasn't a good single place to put the decrement. There are so many ways to throw an error or return a value.